pkg/cmd/cli/cmd/loginoptions.go

```diff
@@ -159,6 +159,13 @@ func (o *LoginOptions) getClientConfig() (*restclient.Config, error) {
 				fmt.Fprintln(o.Out)
 			}
 
+		// path "/" we tried to ping is forbidden, but the server is still reachable
+		case clientcmd.IsForbidden(result.Error()):
```

Comment:
We will probably remove /oapi at some point in the future.  Use `/apis`,
that is more future proof.  Well.  For now.

On Mon, Jun 20, 2016 at 3:50 PM, Fabiano Franz <notifications@github.com>
wrote:

> In pkg/cmd/cli/cmd/loginoptions.go
> <https://github.com/openshift/origin/pull/9413#discussion_r67754274>:
>
> > @@ -159,6 +159,13 @@ func (o *LoginOptions) getClientConfig() (*restclient.Config, error) {
> >  				fmt.Fprintln(o.Out)
> >  			}
> >
> > +		// path "/" we tried to ping is forbidden, but the server is still reachable
> > +		case clientcmd.IsForbidden(result.Error()):
>
> @smarterclayton <https://github.com/smarterclayton> Updated to ping the
> root of OpenShift API set in config.APIPath (/oapi at the moment).
> @liggitt <https://github.com/liggitt> Removed the IsForbidden check. PTAL.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9413/files/751e12702dd0ccabd2ee55090978c067dc7e6ff3#r67754274>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p1eQecXV2czF1HZhfW052ZYgb-Zcks5qNu79gaJpZM4I4yLF>
> .
>